### ISSUE TITLE
Language change not marked on print modal

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4087]
## Area or Component
Results page - print modal

## Quick Summary
There's a button at the top of the modal for "Espanol", to get the results in Spanish. This language change isn't encoded into the application, so screenreaders will read "Espanol" with an English/American pronunciation.

## Accessibility Violation
Change in language is not marked: https://cdc-devtools.dequecloud.com/rule-help/en/lang-change-not-marked

## Action Requested
Code "Espanol" button as Spanish-language.

## Screenshots (optional)
![Screen Shot 2022-07-26 at 4 29 33 PM](https://user-images.githubusercontent.com/80282552/181129426-1f8bb7de-b789-49c4-a071-0124957cb82d.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=1978534823&range=26:26
